### DISCUSSION TITLE
Add lights to Technic Top Gear Rally Car (42109) with the Powered Up Remote

### DISCUSSION THREAD

**Discussion by Caovgt** [pybricks/support#1968] [Q&A]
Hi everyone, I successfully installed Pybricks to the Rally car to drive with the Powered Up remote. 
Now I'm trying to add some lights. I want to set them On when I push a button and set them OFF when I push the button again but I don't find the right coding. At the moment I set it like this. The lights go ON when I push any button.
How could I solve this.
Thx
<img width="993" alt="Screenshot 2024-12-08 at 11 00 28" src="https://github.com/user-attachments/assets/cb9f06d1-5e77-484c-9012-6bd4affd676b">

**Comment by BertLindeman** [comment]
Hello,

As an example I added a variable as a boolean that switches True / False if the left red button is pressed.
Then a bit of a wait for the button to be released.

Looks like this:
![pybricks_program_2024-12-08T13_53_57](https://github.com/user-attachments/assets/036f3550-8ce9-4ecb-9e9c-09a8a0471c76)


Bert

**Reply by Caovgt** [reply]
Hi Bert, thank you very much.
